### PR TITLE
[BHE-2020] Create Event DevopsDaysBH 2020 and include the date.

### PR DESCRIPTION
**Pull request by danielreisp**
Hello! 
We created the event DevopsDaysBH 2020, including team member and the date 26/09/2020.
[]'s.
Daniel

### CODE REVIEW

File: data/events/2020-belo-horizonte.yml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+description: "Devopsdays is coming to belo-horizonte!" # Edit this to suit your preferences
+ga_tracking_id: "UA-124055941-1" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2020-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2020-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: 2020-09-26T00:00:00-03:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2020-09-26T23:59:59-03:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
```
Comment (somatorio) [inline data/events/2020-belo-horizonte.yml]: Do you already have the venue reserved? If not, please remove the date as specified 
```
Leave blank if you don't have a venue reserved yet.
```

File: data/events/2020-belo-horizonte.yml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Aline M. Lima"
+    linkedin: "https://www.linkedin.com/in/alinemarinholima"
+    bio: ""
+    image: "aline-marinho-lima.jpg"
```
Comment (somatorio) [inline data/events/2020-belo-horizonte.yml]: We need you to send an email to info@devopsdays.org about the team changes with the name and email of the new members and with the ones to be removed, so we can sync the event email list and slack

File: data/events/2020-belo-horizonte.yml (hunk 1 above)
Comment (danielreisp) [reply data/events/2020-belo-horizonte.yml]: Hello Sum! Thanks for your attention, we chose Puc Minas for this moment ok? We already have a partnership with PUC Minas.

File: data/events/2020-belo-horizonte.yml (hunk 2 above)
Comment (bridgetkromhout) [reply data/events/2020-belo-horizonte.yml]: I have updated email and Slack with the new members.

**Comment by bridgetkromhout** [comment]
The appveyor build failure is not due to issues in this PR.

**Pull request merged**